File: src/Cores.java
import java.util.Scanner;

public class Cores {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Pedir ao usuário para inserir dois números
        System.out.println("Digite o primeiro número:");
        int num1 = scanner.nextInt();
        System.out.println("Digite o segundo número:");
        int num2 = scanner.nextInt();

        // Verificar se ambos os números são positivos
        if (num1 > 0 && num2 > 0) {
            System.out.println("Azul");
        }
        // Verificar se ambos os números são ímpares
        else if (num1 % 2 != 0 && num2 % 2 != 0) {
            System.out.println("Laranja");
        }
        // Caso contrário, imprimir "Roxo"
        else {
            System.out.println("Roxo");
        }

        scanner.close();
    }
}

File: src/cores.js
// Importar o módulo readline para ler a entrada do usuário
const readline = require('readline');

// Configurar o readline para usar a entrada e saída padrão
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

// Função para verificar se um número é ímpar
function isOdd(num) {
    return num % 2 !== 0;
}

// Pedir ao usuário para inserir dois números
console.log("Digite o primeiro número:");
rl.question('', (input1) => {
    console.log("Digite o segundo número:");
    rl.question('', (input2) => {
        // Converter as entradas para números inteiros
        let num1 = parseInt(input1);
        let num2 = parseInt(input2);

        // Verificar se ambos os números são pares
        if (num1 % 2 === 0 && num2 % 2 === 0) {
            console.log("Azul");
        }
        // Verificar se ambos os números são ímpares
        else if (isOdd(num1) && isOdd(num2)) {
            console.log("Laranja");
        }
        // Caso contrário, imprimir "Roxo"
        else {
            console.log("Roxo");
        }

        // Fechar a interface readline
        rl.close();
    });
});
